src/antigravity/memory.py

```
@@ -0,0 +1,259 @@
+        # Ensure storage directory exists
+        self.storage_path.mkdir(parents=True, exist_ok=True)
+
+        logger.info(
+            "Memory manager initialized",
+            max_tokens=self.max_tokens,
+            threshold=self.summarization_threshold
+        )
+
+    def _generate_id(self, content: str) -> str:
+        """Generate unique ID for entry."""
+        timestamp = datetime.now().isoformat()
```

Ruff (0.14.8)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)